packages/plugin-core/src/features/completionProvider.ts

```diff
@@ -105,7 +105,7 @@ export const provideCompletionItems = sentryReportingCallback(
     }
 
     const line = document.lineAt(position).text;
-    Logger.debug({ ctx, position, msg: "enter" });
```

Comment:
Did you mean to leave this change in?  This is pretty noisy right?

Reply:
yeah - its helpful to debug. completion provider only gets triggered on triggers so not that noisy. but will switch back to debug